Planner/src/User.java

```diff
@@ -5,6 +5,7 @@
 public class User {
 	String name;
 	String surname;
+	ArrayList<User> assignet_to;
```

Comment:
The assigned_to should be on Task. A Task is assigned to a user. Please move this to Task. The member should be User assigned_to.